helm/platform-health/templates/role.yaml

```diff
@@ -0,0 +1,63 @@
+---
+{{- $role := .Values.role | default dict }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
```

Comment:
I suspect that the common case (validate resources and Helm charts across more than deployment namespace) would require `ClusterRole` rather than `Role`, but that should probably be configured via values.

Reply:
Right. I wrongly assumed that one would have a satellite per namespace. I will draft a ClusterRole+Binding.